cmake/symlink_install/catkin_symlink_install_directory.cmake

```diff
@@ -72,7 +72,7 @@ function(catkin_symlink_install_directory directory_keyword)
 
     string(REPLACE ";" "\" \"" argn_quoted "\"${argn}\"")
     catkin_symlink_install_append_install_code(
-      "catkin_symlink_install_directory(DIRECTORY ${argn_quoted})"
+      "catkin_symlink_install_directory(CURRENT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR} DIRECTORY ${argn_quoted})"
```

Comment:
Please quote the value to make sure spaces in the path are still passed as a single argument.

Same below.

Reply:
Quote to avoid variable expansion in 418b9d08478e6a65a4780eda5a12b73bddd09313